ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less

```diff
@@ -78,17 +78,17 @@
   .dropdown-text {
     padding: 0px 10px;
     width: ~'calc(100% - @{minimumButtonSize})';
-    min-height: @minimumButtonSize;
+    height: @minimumButtonSize;
```

Comment:
When multiple sources are selected, they are separated by ` | ` and will expand to the next line if needed. I think this change will remove the ability for the list of selected sources to continue on the next line.